protobuf/types/service.proto

```diff
@@ -1,68 +1,229 @@
 syntax = "proto3";
 
-package types;
-option go_package = "github.com/mesg-foundation/engine/protobuf/types";
+import "gogo/protobuf/gogoproto/gogo.proto";
+
+package service;
```

Comment:
the proto package doesn't need to change. only the go_package need to change 👍 
```suggestion
package types;
```
like this the file `protobuf/api/service.proto` doesn't need any modification

Reply:
I don't liek the idea of different package names in different context, this is why I think it's better to have the same package name.

Reply:
this have some impact on mesg-js. and again we are speaking about proto package, not go.
the generated types are different now
```diff
-            tasks?: (types.Service.ITask[]|null);
+            tasks?: (service.Service.ITask[]|null);
```

Reply:
yes I know and it's only one line of code and I still think they should stay the same (both types or service)